src/platform/packages/shared/shared-ux/chrome/navigation/src/ui/components/panel/navigation_panel.tsx

```diff
@@ -13,14 +13,13 @@ import {
   EuiPanel,
   EuiWindowEvent,
   keys,
-  useEuiTheme,
 } from '@elastic/eui';
 import React, { useCallback, type FC } from 'react';
 import classNames from 'classnames';
 
 import type { PanelSelectedNode } from '@kbn/core-chrome-browser';
 import { usePanel } from './context';
-import { getNavPanelStyles, getPanelWrapperStyles } from './styles';
+import { navPanelStyles, panelWrapperStyles } from './styles';
```

Comment:
Initially, I thought the bug was caused by styling, so I cleaned up styles in this component to use `@emotion/react` instead of `@emotion/css`.